src/main/java/io/confluent/connect/jdbc/JdbcUtils.java

```diff
@@ -131,7 +131,8 @@ public static String getAutoincrementColumn(Connection conn, String table) throw
     log.trace("Falling back to SELECT detection of auto-increment column for {}:{}", conn, table);
     Statement stmt = conn.createStatement();
     try {
-      rs = stmt.executeQuery("SELECT * FROM \"" + table + "\" LIMIT 1");
+      String quoteString = getIdentifierQuoteString(conn);
+      rs = stmt.executeQuery("SELECT * FROM " + quoteString + table + quoteString + " LIMIT 1");
```

Comment:
You don't need to worry about SQL injection at all, right? All these values like table name and timestamp.column.name are coming from config files, which you set yourself, so the worst you can do is SQL inject against yourself, likely against against a MySQL server that you already have permissions for?

Relevant XKCD post: https://xkcd.com/327/